integration-cli/docker_cli_wait_test.go

```diff
@@ -77,16 +69,14 @@ func (s *DockerSuite) TestWaitBlockedExitRandom(c *check.C) {
 	testRequires(c, DaemonIsLinux)
 	out, _ := dockerCmd(c, "run", "-d", "busybox", "/bin/sh", "-c", "trap 'exit 99' TERM; while true; do usleep 10; done")
 	containerID := strings.TrimSpace(out)
-	c.Assert(waitRun(containerID), check.IsNil)
+	c.Assert(waitRun(containerID), checker.IsNil)
 
 	chWait := make(chan error)
 	waitCmd := exec.Command(dockerBinary, "wait", containerID)
 	waitCmdOut := bytes.NewBuffer(nil)
 	waitCmd.Stdout = waitCmdOut
-	if err := waitCmd.Start(); err != nil {
-		c.Fatal(err)
-	}
-
+	err := waitCmd.Start()
+	c.Assert(err, checker.IsNil)
```

Comment:
this can be merge into c.Assert(waitCmd.Start(), checker.IsNil), but I don't think statement like  

        err := waitInspect(containerID, "{{.State.Running}}", "false", 30*time.Second)
        c.Assert(err, checker.IsNil) //Container should have stopped by now